scripts.go

```diff
@@ -1,3 +1,48 @@
-//go:generate protoc --go_out=. protobuf/stream.proto
+//go:generate go run scripts.go
 
 package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	goPath = os.Getenv("GOPATH")
+)
+
+func main() {
+	if err := generateProtos("."); err != nil {
+		fmt.Printf("%+v", err)
+	}
+}
+
+func generateProtos(dir string) error {
+
+	return filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+		// skip vendor directory
+		if info.IsDir() && info.Name() == "vendor" {
+			return filepath.SkipDir
+		}
+		// find all protobuf files
+		if filepath.Ext(path) == ".proto" {
+			// args
+			args := []string{
+				"-I=.",
+				fmt.Sprintf("-I=%s/src", goPath),
```

Comment:
If you want this to be cross platform supported, can use `filepath.Join()` to get the right separators on windows, linux.
https://golang.org/pkg/path/filepath/#Join

Reply:
right forgot, will add it to my PR with more tests